usagov/usagov-2021
# Usagov 1614 breadcrumb spacing

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1614

## Description

Reduce space above and below the breadcrumb
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


This test is mainly an optical test. On desktop, inspect any page, not the homepage, and verify the the html tag "main" has a padding-top: 1.7rem; and the "`<div>`" tag above the "<div id="block-usagov-breadcrumbs" ...> has a height: 3rem;
This spacing should look the same in tablet and mobile although the values will change.

<img width="1512" alt="Screenshot 2024-08-05 at 2 41 14 PM" src="https://github.com/user-attachments/assets/15a5f4f9-23e0-4e62-bc5d-b56f430e32c2">

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
